src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
+
+    protected String dueDate;
+
+    public Deadline(String description, String dueDate) {
+        super(description);
+        this.dueDate = dueDate;
+    }
+
+    @Override
+    public String toString(){
```

Comment:
```suggestion
    public String toString() {
```
Perhaps add a space here between the `()` and `{`? I noticed the same issue in multiple other places too.